common/transform/src/test/java/ai/h2o/mojos/deploy/common/transform/MojoScorerTest.java

```diff
@@ -136,35 +134,326 @@ void verifyScoreRequestWithOriginalShapley_fails() {
     given(scoreResponseConverter.apply(any(), any()))
             .willReturn(dummyResponse);
 
+    MojoScorer scorer = dummyScorer();
+
     // When & Then
     assertThrows(
             IllegalArgumentException.class, () -> scorer.score(request));
   }
 
   @Test
-  void verifyContributionWithOriginalShapley_fails() {
+  void verifyContributionWithOriginalShapley_ShapleyDisabled_Fails() {
+    // Given
+    System.setProperty("shapley.enable", "false");
     ContributionRequest request = new ContributionRequest();
     request.addFieldsItem("field1");
     request.addRowsItem(toRow("text"));
     request.setRequestShapleyValueType(ShapleyType.ORIGINAL);
 
+    MojoScorer scorer = dummyScorer();
+
     // When & Then
     assertThrows(IllegalArgumentException.class, () -> scorer
             .computeContribution(request));
   }
 
   @Test
-  void verifyContributionWithTransformedShapley_fails() {
+  void verifyContributionWithTransformedShapley_ShapleyDisabled_Fails() {
+    // Given
     ContributionRequest request = new ContributionRequest();
     request.addFieldsItem("field1");
     request.addRowsItem(toRow("text"));
     request.setRequestShapleyValueType(ShapleyType.TRANSFORMED);
 
+    MojoScorer scorer = dummyScorer();
+
     // When & Then
     assertThrows(IllegalArgumentException.class, () -> scorer
             .computeContribution(request));
   }
 
+  @Test
+  void verifyScoreRequestWithoutShapley_ShapleyEnabled_Succeeds() {
+    // Given
+    System.setProperty("shapley.enable", "true");
+
+    ScoreRequest request = new ScoreRequest();
+    request.addFieldsItem("field1");
+    request.addRowsItem(toRow("text"));
+    MojoFrame dummyMojoFrame = generateDummyTransformedMojoFrame();
+    given(scoreRequestConverter.apply(any(), any()))
+        .willReturn(dummyMojoFrame);
+    ScoreResponse dummyResponse = generateDummyResponse();
+    given(scoreResponseConverter.apply(any(), any()))
+        .willReturn(dummyResponse);
+
+    MojoScorer scorer = dummyScorer();
+
+    // When
+    scorer.score(request);
+
+    // Then all ok
```

Comment:
Do we know all is okay because no exception was thrown by scorer.score()?  If so, then asserting with something like:
```
// When & Then
assertDoesNotThrow(...)
```

Reply:
... well at the moment yes.

Reply:
tbh I noticed we kind of consume the exception if anything DOES happen... which is kind of not ok.